app/Http/Controllers/Backend/AdminUserController.php

```diff
@@ -75,10 +70,16 @@ public function index()
      *
      * @return \Illuminate\Http\Response
      */
-    /*public function update(Request $request, $id)
+    public function update(AdminUserEditRequest $request, $id)
     {
-        //
-    }*/
+        $adminUser = AdminUser::findOrFail($id);
+        $adminUser->name = $request->get('name');
```

Comment:
No need to assign attribute one by one. Just use `fill` method or `update` method
```php
$adminUser->fill($request->all());
$adminUser->save();
#OR
$adminUser->update($request->all());